pkg/fleetautoscalers/controller.go

```diff
@@ -293,3 +324,62 @@ func (c *Controller) updateStatusUnableToScale(ctx context.Context, fas *autosca
 
 	return nil
 }
+
+// createFasThread creates a FleetAutoScaler sync routine
+func (c *Controller) createFasThread(ctx context.Context, fas *autoscalingv1.FleetAutoscaler) error {
+	key := fas.Namespace + "/" + fas.Name
```

Comment:
To be consistent, we can use `cache.MetaNamespaceKeyFunc(fas)` to get the key in the right format that it is used all the way through. Or maybe just pass the key through? You have it already in the parent functions it seems.

Reply:
Updated to `cache.MetaNamespaceKeyFunc(fas)`. This can save a parameter and make sure the key won't be inconsistent with `fas` itself. Otherwise, I will have to call this in the AddEventHandler code, not many codes saved.